router/worker.go

```diff
@@ -426,6 +426,7 @@ func (w *worker) processDestinationJobs() {
 					if err != nil {
 						errorAt = routerutils.ERROR_AT_TF
 						respStatusCode, respBody = types.RouterUnMarshalErrorCode, fmt.Errorf("transformer response unmarshal error: %w", err).Error()
+						w.logger.Errorf("transformer response unmarshal error: %v, destType: %s, destID: %s, jobID: %d", err, w.rt.destType, destinationID, destinationJob.JobMetadataArray[0].JobID)
```

Comment:
Does `JobMetadataArray[0]` always exist when reaching this line?

Reply:
Some extreme cases, where in the metadata is being sent as an object(mistkenly) then `JobMetadataArray[0]` wouldn't exist. Hence removing it